utils/src/main/kotlin/Utils.kt

```diff
@@ -67,6 +67,29 @@ val ROOT_LICENSE_FILENAMES = listOf(
     "readme*"
 ).generateCapitalizationVariants()
 
+/**
```

Comment:
commit-message: Is the grouping really about `T` or is it about `Any` primarily?

Reply:
To me, it's really about the `T`in `T.fixupUserHomeProperty()`, as for sorting I regard the `T` to be part of the function name. That is, I'd sort `T.fixupUserHomeProperty()` after `Any.fixupUserHomeProperty()` (if we had both) because `T` comes after `A`.